tests/default_vars.sh

```diff
@@ -821,8 +821,9 @@ export DO_CA=.true.
 export CA_SGS=.true.
 export CA_GLOBAL=.false.
 export NCA=1
-export SCELLS=2600
-export TLIVES=1800
+export NCELLS=5
+export NLIVES=12
+export NTHRESH=18
```

Comment:
Can we remove these settings from ``default_vars``? 

```
# CA tunable parameter
export RCELL_c48=0.1
export RCELL_c96=0.1
export RCELL_c192=0.2
export RCELL_c384=0.72
export RCELL_c768=0.72
```

Reply:
Yes, thanks. I will remove these.